oracle/cpi-config.yml

```diff
@@ -0,0 +1,10 @@
+cpis:
+- name: oracle-cpi-ref
+  type: oracle  #CPI binary name. Director appends _cpi to this name
```

Comment:
This comment may confuse users trying to review the ops. Do you find it helpful?

Reply:
Nah, not really.  That comment was added when w were figuring things out. Will remove it.